azext_iot/operations/central.py

```diff
@@ -0,0 +1,65 @@
+from knack.util import CLIError
+from azext_iot._factory import _bind_sdk
+from azext_iot.common._azure import (get_iot_hub_token_from_central_app_id, get_event_hub_target_from_central_app_id)
+from azext_iot.common.shared import SdkType
+from azext_iot.common.utility import (unpack_msrest_error)
+from azext_iot.common.basic_sas_token_auth import BasicSasTokenAuthentication
+
+
+def find_between(s, start, end):
+    return (s.split(start))[1].split(end)[0]
+
+
+def iot_central_device_show(cmd, device_id, app_id, aad_token=None):
+    sasToken = get_iot_hub_token_from_central_app_id(cmd, app_id, aad_token)
+    endpoint = find_between(sasToken, 'SharedAccessSignature sr=', '&sig=')
+    target = {'entity': endpoint}
+    auth = BasicSasTokenAuthentication(sas_token=sasToken)
+    service_sdk, errors = _bind_sdk(target, SdkType.service_sdk, auth=auth)
+    try:
+        return service_sdk.get_twin(device_id)
+    except errors.CloudError as e:
+        raise CLIError(unpack_msrest_error(e))
+
+
+def iot_central_monitor_events(cmd, app_id, device_id=None, consumer_group='$Default', timeout=300, enqueued_time=None,
```

Comment:
This command uses a lot of the same "setup" elements from iot_hub_monitor_events. Consider combining these elements in a reusable way (i.e. something like `def _init_monitoring(...):`